src/imitation/rewards/serialize.py

```diff
@@ -173,16 +173,21 @@ def _validate_wrapper_structure(
 def load_zero(path: str, venv: VecEnv) -> reward_function.RewardFn:
     del path, venv
 
-    def f(
-        obs: np.ndarray,
-        act: np.ndarray,
-        next_obs: np.ndarray,
-        dones: np.ndarray,
-    ) -> np.ndarray:
-        del act, next_obs, dones  # Unused.
-        return np.zeros(obs.shape[0])
-
-    return f
+    class f(reward_function.RewardFn):
```

Comment:
Do we need to inherit? `reward_function.RewardFn` is a protocol so I think a function with the type-signature should match too (perhaps not?)

Reply:
Ah, yes it seems like it's sufficient to rename the argument names. I would not have thought that argument names were a type enforced part of a Protocol!